integration-cli/docker_utils.go

```diff
@@ -1405,6 +1405,12 @@ func waitRun(contID string) error {
 	return waitInspect(contID, "{{.State.Running}}", "true", 5)
 }
 
+// waitExited will wait for the specified container to state exit, subject
+// to a maximum time limit in seconds supplied by the caller
+func waitExited(contID string, seconds int) error {
```

Comment:
I used the same signature as the existing waitInspect.... Do you want both changed and the callers?